templates/workbooks/workbook_plot.html

```diff
@@ -40,7 +40,7 @@
         </div>
 
         <!-- Settings Flyout -->
-        <div class="fly-out settings-flyout plot-config" style="overflow-y: scroll;" id="plot-{{ worksheet.id }}-settings">
+        <div class="fly-out settings-flyout plot-config" style="overflow-y: auto;" id="plot-{{ worksheet.id }}-settings">
```

Comment:
I'm fine with the display:hidden styles staying inline. All others should be folded in. A new issue sounds good, but this should be lower priority than other bugs. Thanks!